src/main/java/com/example/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package com.example.configuration;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfiguration {
+
+    @Bean
+    @ConditionalOnProperty(name = "enableThisIsMyFirstConditionalBean", havingValue = "true")
+    public String ThisIsMyFirstConditionalBean() {
+        return "Conditional Bean is enabled!";
+    }
```

Comment:
when you create bean, then you need not just create a method with some condition, you also need to return this been, not a `String`:
```java
    @Bean
    @ConditionalOnProperty(name = "app.allowConditionalBean", havingValue = "true")
    public ThisIsMyFirstConditionalBean thisIsMyFirstConditionalBean() {
        return new ThisIsMyFirstConditionalBean();
    }
```

Reply:
+